assignments/callbacks.js

```diff
@@ -31,4 +37,10 @@ function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
-}
+  let duplicateFree = [];
```

Comment:
Nice job with this stretch problem! I think this is actually a better solution than the model.